lms/djangoapps/certificates/tests/test_api.py

```diff
@@ -92,7 +93,8 @@ def setUp(self):
         self.course = CourseFactory.create(
             org='edx',
             number='verified',
-            display_name='Verified Course'
+            display_name='Verified Course',
+            end=datetime.datetime.now()
```

Comment:
Just another FYI, I didn't add a test here because I moved the logic out of the API and into the views.